packages/thirdweb/src/wallets/smart/lib/userop.ts

```diff
@@ -119,7 +119,8 @@ export async function createUnsignedUserOp(args: {
         userOp: partialOp,
         options,
       });
-      partialOp.callGasLimit = estimates.callGasLimit;
+      // add 50k overhead for entrypoint checks (following same as unity-sdk)
+      partialOp.callGasLimit = estimates.callGasLimit + 50_000n;
```

Comment:
missing one case line 113.

also can you remove `(following same as unity-sdk)` please